interface/src/app/analyze/analysis/sampleBin.js

```diff
@@ -30,7 +30,7 @@ angular.module('adage.analyze.sampleBin', [
 function($log, $cacheFactory, $q, Sample, Activity, Signature, SignatureSet,
 MathFuncts, errGen) {
   var SampleBin = {
-    selectedMlModel: {
+    mlModelInfo: {
```

Comment:
Gotcha, that makes sense, if we are keeping track of it in that issue or another one that gets broken out like you say.